syntax = "proto3";
package dbinstance;




import "google/protobuf/empty.proto";

/*
DeleteDBInstance请求
*/
message DeleteDBInstanceRequest {
    /*
    数据库实例ID
    */
    string dbInstanceId = 1 ;
    /*
    数据库服务ID
    */
    string dbServiceId = 2 ;
}

/*
DeleteDBInstanceApi返回
*/
message DeleteDBInstanceResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

